nextflow_schema.json

```
@@ -5,28 +5,12 @@
     "description": "Pipeline for the analysis of crispr data",
     "type": "object",
     "definitions": {
-        "alignment_parameters": {
-            "title": "Alignment parameters",
-            "type": "object",
-            "description": "",
-            "default": "",
-            "properties": {
-                "aligner": {
-                    "type": "string",
-                    "description": "Aligner program to use",
-                    "default": "minimap2",
-                    "fa_icon": "fas fa-align-justify",
-                    "enum": ["minimap2", "bwa", "bowtie2"]
-                }
-            },
-            "required": ["aligner"]
-        },
         "input_output_options": {
             "title": "Input/output options",
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir"],
+            "required": ["outdir", "analysis"],
```

Isn't the input also required? (except for running tests)
```suggestion
            "required": ["input", "outdir", "analysis"],
```